CMP1903_A1_2324/Testing.cs

```diff
@@ -8,13 +9,37 @@ namespace CMP1903_A1_2324
 {
     internal class Testing
     {
-        /*
-         * This class should test the Game and the Die class.
-         * Create a Game object, call the methods and compare their output to expected output.
-         * Create a Die object and call its method.
-         * Use debug.assert() to make the comparisons and tests.
-         */
+     
+        public void GameTest() //GameTest method
+        {
+            //creates a game object
+            Game game = new Game();
+            Console.WriteLine("TestGame");
+            int test = game.Start(); //this uses the game class method
 
-        //Method
+
+            Debug.Assert(test > 18 || test>=3,"Game test unsuccessful"); // compares the outcome of the game and check if the total is
+            //between 3-18, if otherwise the assertation catches it
```

Comment:
The condition in the Debug.Assert allows any number bigger than 3 to enter
I'd personality change from "test > 18 || test>=3" to "test <= 18 || test>=3"